File: Difficulty: Easy/Segregate 0s and 1s/segregate-0s-and-1s.java
class Solution {
    void segregate0and1(int[] arr) {
        // code here
        int i=0;
        for(int index=0;index<arr.length;index++){
            if(arr[index]!=1){
                int temp=arr[index];
                arr[index]=arr[i];
                arr[i]=temp;
                i++;
            }
        }
    }
}

File: Difficulty: Easy/Segregate 0s and 1s/segregate-0s-and-1s.cpp
class Solution {
  public:
    void segregate0and1(vector<int> &arr) {
        // code here
        int i=0;
        int j=arr.size()-1;
        int index=0;
        while(index<arr.size()){
            if(arr[index]==0){
                swap(arr[i++],arr[index]);
            }
            index++;
        }
        return;
    }
};